Update FAQ-UseWasabi.md (#1275)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

### How can I change the label of my receive address?

You can change the label of your unused receive address by clicking on the edit icon.

> This is useful when you have generated a receiving address with a specific label, but then the sender has changed.

Reviewer: ```suggestion
This is useful when you have generated a receiving address with a specific label, but then the sender (anyone that knows this address is yours) has changed.
```

> Take care with whom you have shared this address, because if you send it to several people, they might all send many coins to the same address.

Reviewer: ```suggestion
Take care with whom you have shared this address, because if you send it to several people, they all know this address belongs to you, and they might all send many coins to the same address.
```

There is a risk that both of them send coins to this same destination, thus unnecessarily linking the payments.

:::danger
It is especially important to **NEVER** send your extended public key to any third party server.
It is especially important to **NEVER** send your extended public key to any third party.
This is a complete de-anonymization of your entire wallet!!
:::
::::

:::details
### Why does Wasabi only use SegWit bech32 addresses?

Wasabi was created after the activation of SegWit, and it makes sense to support the most advanced address type, which has numerous benefits.

> For example, due to the malleability fix of SegWit, you can now remix CoinJoin outputs which are currently unconfirmed.

Reviewer: Why was this example removed?

Author: Because I can't remix coinjoin outputs which are currently unconfirmed. At least not indefinitely.
![image](https://user-images.githubusercontent.com/51679301/194728194-8d836682-e7a3-44fe-b213-5165954efbe2.png)

Reviewer: I think you can.
That messages is probably because you have banned coins.

Author: No banned coins. Just a long chain of coinjoins.

Reviewer: AKAIK we do not have anything in the protocol (or the backend configurations) that prevents unconfirmed coinjoin outputs from remixing.
@MaxHillebrand WDYT?

Author: I'll leave this section out of this PR

> Implementing Taproot hasn't been a priority, as the benefits of upgrading to it are negligible for Wasabi, at the moment.

Reviewer: But we are currently working on implementing Taproot.

Author: I'll leave this section out of this PR

> Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits, like large savings on mining fees for SegWit transactions.
> Implementing Taproot hasn't been a priority, as the benefits of upgrading to it are negligible for Wasabi, at the moment.

Author: Ok I'll leave this section out of this PR
```suggestion
Wasabi was created after the activation of SegWit, and it makes sense to support the most advanced address type, which has numerous benefits.
For example, due to the malleability fix of SegWit, you can now remix CoinJoin outputs which are currently unconfirmed.
There are also large savings on mining fees for SegWit transactions.
```

There is a risk that both of them send coins to this same destination, thus unnecessarily linking the payments.

:::danger

> It is especially important to **NEVER** send your extended public key to any third party.

Reviewer: ```suggestion
It is especially important to **NEVER** send your extended public key to any third party (anyone).
```

:::

:::details

> ### Where can I find my address QR code and the corresponding QR code?

Reviewer: What is the difference?

Author: Good question. I prolly got confused while looking at 3 versions of the same file.

> ### Where can I find my address QR code and the corresponding QR code?

Author: ```suggestion
### Where can I find my address QR code?
```
